packages/controller/lib/scripts/scripts.js

```diff
@@ -83,7 +84,7 @@ if (process.argv.indexOf('--prepublish') !== -1) {
             console.error(`Cannot read sources file "${stableURL}": ${err}`);
             process.exit(2);
         } else {
-            fs.writeFileSync(`${__dirname}/../../conf/sources-dist.json`, body);
+            fs.writeFileSync(path.normalize(`${tools.getControllerDir()}/conf/sources-dist.json`), body);
```

Comment:
```suggestion
            fs.writeFileSync(path.join(tools.getControllerDir(), 'conf/sources-dist.json'), body);
```
would be more consistent with the other locations